moira/db.py

```diff
@@ -1072,6 +1074,25 @@ def getEvents(self, trigger_id=None, start=0, end=100):
         defer.returnValue([anyjson.deserialize(e) for e in events])
 
     @defer.inlineCallbacks
+    @docstring_parameters(TARGETS)
+    def getTargets(self):
+        """
+        getTargets(self)
+
+        Returns all targets from db (variant = False)
+        or returns list targets in set TARGETS
+
+        :rtype: list
+        """
+        variant = False
+        if variant:
+            targets = yield self.rc.hkeys(TARGETS)
+            defer.returnValue(targets)
+        else:
+            targets = yield self.rc.keys("moira-metric-data*")
```

Comment:
Also, you should use constants like `METRIC_PREFIX` instead of hardcoding `moira-metric-data` key name.